### ISSUE TITLE
loadBalancerIP is ignored in azure mode

### ISSUE THREAD

**Issue by jalberto** [appscode/voyager#572]
k8s: 1.7.2
voyager: 3.2.1

When create an ingress with annotation `ingress.appscode.com/load-balancer-ip: x.x.x.x`the service is created, but without static IP (get a random public IP).

The IP I use is already created in azure and ready to be used. A regular service with `loadBalancerIP` works as expected.

**Comment by tamalsaha** [comment]
@jalberto , can you show up the Service created by voyage operator for the ingress? It should be named `voyager-{ingress-name}` and in the same namespace as the Ingress.

**Comment by jalberto** [comment]
@tamalsaha I am using helm chart in 3.2.1 tag

```
{
  "kind": "Service",
  "apiVersion": "v1",
  "metadata": {
    "name": "lb-voyager",
    "namespace": "default",
    "selfLink": "/api/v1/namespaces/default/services/lb-voyager",
    "uid": "e5e04ce6-aa10-11e7-9cc7-000d3a2b1d50",
    "resourceVersion": "1226553",
    "creationTimestamp": "2017-10-05T21:05:21Z",
    "labels": {
      "app": "voyager",
      "chart": "voyager-1.1.0",
      "heritage": "Tiller",
      "release": "lb"
    }
  },
  "spec": {
    "ports": [
      {
        "name": "http",
        "protocol": "TCP",
        "port": 56790,
        "targetPort": "http"
      }
    ],
    "selector": {
      "app": "voyager",
      "release": "lb"
    },
    "clusterIP": "10.0.124.129",
    "type": "ClusterIP",
    "sessionAffinity": "None"
  },
  "status": {
    "loadBalancer": {}
  }
}
```

**Comment by tamalsaha** [comment]
First question is, if you deployed Voyager via Helm , did you use the stable chart? or did you use the chart from this repo?

**Comment by tamalsaha** [comment]
Also, if you want interactive response, you can find me tamal@ on our slack channel https://slack.appscode.com/

**Comment by jalberto** [comment]
```
apiVersion: v1
kind: Service
metadata:
  annotations:
    ingress.appscode.com/origin-api-schema: voyager.appscode.com/v1beta1
    ingress.appscode.com/origin-name: haproxy
  creationTimestamp: 2017-10-05T21:09:58Z
  name: voyager-haproxy
  namespace: default
  resourceVersion: "1227217"
  selfLink: /api/v1/namespaces/default/services/voyager-haproxy
  uid: 8b117518-aa11-11e7-9ed5-000d3a2b1715
spec:
  clusterIP: 10.0.222.111
  externalTrafficPolicy: Cluster
  ports:
  - name: tcp-80
    nodePort: 31082
    port: 80
    protocol: TCP
    targetPort: 80
  selector:
    origin: voyager
    origin-api-group: voyager.appscode.com
    origin-api-version: v1beta1
    origin-name: haproxy
  sessionAffinity: None
  type: LoadBalancer
status:
  loadBalancer:
    ingress:
    - ip: y.y.y.y
```

**Comment by tamalsaha** [comment]
This fix is included in 4.0.0-rc.5

**Comment by jalberto** [comment]
Thanks @tamalsaha
